sdcm/cluster.py

```diff
@@ -3192,6 +3192,18 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
             self.run_node_benchmarks()
         self.coredumps = {}
         super().__init__()
+        self.keyspace_info = {}
+
+    def get_keyspace_info(self, keyspace_name: str, db_node: BaseNode):
+        # Prevent circular import
```

Comment:
can we fix it, i.e. BaseNode is only used annotations, and we can do the `TYPE_CHECKING` trick

Reply:
changed